### ISSUE TITLE
Configure omission of top and bottom lines from table output

### ISSUE THREAD

**Issue by MidLifeXis** [neilb/Text-Table-Tiny#3]
I am tweaking a data dumper toolset I have, and am looking for a simple formatter to generate table output from an existing AoA into the following form to facilitate pasting into a GitHub comment / issue / etc.

```
|head1|head2|head3|
|-----|-----|-----|
|data1|data2|data3|
|data4|data5|data6|
```

|head1|head2|head3|
|-----|-----|-----|
|data1|data2|data3|
|data4|data5|data6|

I could write this myself (it isn't complex), but if I can utilize prior art, I would prefer it.  :-)

I don't see a method to remove the top and bottom lines in the pod or the source.  Am I missing something?

**Comment by neilb** [comment]
I just released version 0.05 to CPAN, which adds support for a top_and_tail parameter.
This suppresses the top and bottom border lines.

Sorry it took so long!

**Comment by perlancar** [comment]
Since you are producing a Markdown table, perhaps [Text::MarkdownTable](https://metacpan.org/pod/Text::MarkdownTable) is more appropriate here? If you prefer the Text::Table::Tiny interface to produce Markdown, there's a thin glue: [Text::Table::Any](https://metacpan.org/pod/Text::Table::Any). There are also CLI's to make it easy to do it on the command-line. And if you're doing it on the Unix command-line, removing top and bottom line can be done with: `| head -n -1 | tail -n +2`
